### PR TITLE
Install libvirt seeds into image

### PR DESCRIPTION
**Pull request by rustydb**
### Summary and Scope

- Relates to: https://github.com/Cray-HPE/docs-fawkes/pull/2

#### Issue Type

- Bugfix Pull Request
- Docs Pull Request
- RFE Pull Request

- [hypervisor / management-vm] Includes libvirt seeds
- [hypervisor] New isolated network for ncn-h001
- [management-vm] Rename cloud-init file to match underscore naming of existing files
- [management-vm] `ll` alias fix for non-root users
- [all] Adds `lsblk` to the list of installed programs in the initrd (since no custom dracut is installed in the hypervisor, nothing was bringing `lsblk` in anymore and I use it frequently in initrds to debug).

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [ ] I have tested this by building an image using this branch HEAD, or this is does not pertain to the node-images pipeline.
- [ ] I tested this on internal system or this pertains to the node-images pipeline (if yes and not build related, please include results or a description of the test)
- [ ] I tested this on a vshasta system or this pertains to the node-images pipeline (if yes and not build related, please include results or a description of the test)

### CODE REVIEW

**Review by heemstra — APPROVED** [review]

File: roles/node_images_hypervisor/files/management-vm/isolated.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<network>
+  <name>isolated</name>
+  <ip address='192.168.254.1' netmask='255.255.255.252'>
+    <dhcp>
+      <range start='192.168.254.2' end='192.168.254.2' />
```
Comment (manderson-hpe) [inline roles/node_images_hypervisor/files/management-vm/isolated.xml:5]: For the management VM, we may want to specify the IP address to use? Are you accomplishing that by setting the range to one IP? I guess that works...

**Review by manderson-hpe — APPROVED** [review]

**Comment by manderson-hpe** [comment]
My only question is more of a comment in regards to anything we do on the `hypervisor` that is magic, we should find a way to make that accessible in the documentation so someone can set this up in a dev environment or in GCP.

File: roles/node_images_hypervisor/files/management-vm/isolated.xml (hunk 1 above)
Comment (rustydb) [reply roles/node_images_hypervisor/files/management-vm/isolated.xml]: That makes sense. I'll do that.

**Comment by rustydb** [comment]
This shows as merged but I backed it out, I merged prematurely.

**Pull request merged**